pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
   flutter: ">=2.0.1"
 
 dependencies:
-  dio: ^4.0.0
+  dio: ^4.0.6
```

Comment:
Can you revert this one? ^4.0.6 is subset of ^4.0.0 so no need to break compatibility with the old versions

Reply:
For more context when you increase version constraint of a package the projects depending on that package might stop building and throw version incompatibility errors so at least for me I prefer to support as oldest version as possible while also maintaining latest version support to make sure users can choose to use any version they want. In our case upgrading to `v4.0.6` doesn't add any improvement since we are not going to consume any api introduced with that version. And setting version to `^4.0.0` will let users to pick any version between `>=4.0.0 <5.0.0`.